timon/webclient/webif1/src/components/Timon.vue

```diff
@@ -61,6 +61,7 @@
           :key="probe"
           :class="{
             err: isErrorState(host.name, probe),
+            err: isTimeoutState(host.name, probe),
```

Comment:
Not sure if this is the right approach. We can do it, but  the inital approach was to stay similar / identical to nagios states.

look for example at: https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/statetypes.html

It is an interesting approach however.

I was suggesting more something to detect whether timon is hanging.

The idea was If Probe age. The last time any probe was updated exceeds a certain value, that the entire timon web page is for example changing background color to red to indicate, that timon might be down.

In our case for example. we know that we have is_up probes with a frequency of 2 minutes.
If we want to be conservative wer coude say,that if Proge Age is greater than 10 minutes (or even 30 minutes) we could indicate, that timon is probably hanging / dead. 

So far I noticed, that under some circumstances  timon can block and no probe at all is executed. So far this was the only issue which ever occured.

To set state to TIMEOUT after 1 h is mostly OK, but there are (will be) some probes, that might only be executed once per day so the logic would be more complicated. (except it would be a parameter)